client/extensions/woocommerce/state/sites/orders/activity-log/selectors.js

```diff
@@ -182,6 +182,11 @@ export const getActivityLogEvents = ( state, orderId, siteId = getSelectedSiteId
 				}
 			}
 
+			// Fall back to `createdDate` if an outdated server does not provide`label_cached`.
```

Comment:
Let's remove this, just make sure the server is released first. We control the server deployment, so no need to keep backwards-compatibility here.

Reply:
Done.